multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -59,6 +69,17 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      command: start
+      enable: true
+      content: |
+        [Service]
```

Comment:
Would prefer that this is `RequiredBy` and `Before` `kubelet.service`, and still enabled but not explicitly started.